lib/screens/movie_details.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+import '../../repository/genre_repo.dart';
+import '../../repository/interface/genre_interface.dart';
+import '../constants/constants.dart';
+import '../model/movie_data.dart';
+import '../widgets/movie_details/details_footer.dart';
+import '../widgets/movie_details/details_likes.dart';
+import '../widgets/movie_details/details_header.dart';
+import '../widgets/movie_details/details_images.dart';
+import '../widgets/movie_details/details_rating.dart';
+
+class DetailScreen extends StatefulWidget {
+  const DetailScreen({super.key});
+
+  static const String routeName = "/movie";
+
+  @override
+  State<StatefulWidget> createState() => _DetailScreenState();
+}
+
+class _DetailScreenState extends State<DetailScreen> {
+  final GenreInterface genreRepo = GenreRepository();
```

Comment:
You're not using this object, please delete it!

Reply:
The object is now used to load the genres of each movie in the detail screen.